### PR TITLE
Greenify

### PR DESCRIPTION
**Pull request by Kirkkt**
[UPDATE] Ping on this one, let's get the master branch green before doing anything else.
@chanian @jakecadams @derrickstaten 
Fixed the testing redness by removing tests for old browsers firefox 35 and mobile safari on iOS 7.0.

Minor: also added deviceName according to the guidelines from https://docs.saucelabs.com/reference/platforms-configurator/#/

### CODE REVIEW

**Comment by derrickstaten** [comment]
This looks okay to me and passes tests. I think @jakecadams should give it a quick look as he's far more web savvy than me

File: dist/build.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ goog.DEPENDENCIES_ENABLED && (goog.included_ = {}, goog.dependencies_ = {pathIsM
   }
 }, goog.importScript_ = function(a, b) {
   (goog.global.CLOSURE_IMPORT_SCRIPT || goog.writeScriptTag_)(a, b) && (goog.dependencies_.written[a] = !0);
-}, goog.IS_OLD_IE_ = !goog.global.atob && goog.global.document && goog.global.document.all, goog.importModule_ = function(a) {
+}, goog.IS_OLD_IE_ = !(goog.global.atob || !goog.global.document || !goog.global.document.all), goog.importModule_ = function(a) {
```
Comment (jakecadams) [inline dist/build.js]: what was happening here that prompted this change?

Comment (Kirkkt) [inline dist/build.js]: no idea, it's auto-generated.

fwiw, it actually didn't change any actual logic

Comment (jakecadams) [inline dist/build.js]: cool. looks like its just an update from the google clojure compiler :+1:

**Comment by jakecadams** [comment]
:+1:

**Pull request merged**